### PR TITLE
[libra framework] Add comments to `create_child_vasp_account`. Add partial specs

### PR DESCRIPTION
**Pull request by tzakian**
Adds comments to the `create_child_vasp_account` along with some partial specs for it.

### CODE REVIEW

File: language/stdlib/modules/LibraAccount.move [hunk 1]
```diff
@@ -123,6 +123,7 @@ module LibraAccount {
     /// Attempting to send funds in (e.g.) LBR to an account that exists, but does not have a
     /// Balance<LBR> resource
     const EPAYEE_CANT_ACCEPT_CURRENCY_TYPE: u64 = 18;
+    const EPAYER_DOESNT_HOLD_CURRENCY: u64 = 19;
```
Comment (sblackshear) [inline language/stdlib/modules/LibraAccount.move:126]: Nice!

File: language/stdlib/transaction_scripts/create_child_vasp_account.move [hunk 2]
```diff
@@ -29,4 +46,18 @@ fun create_child_vasp_account<CoinType>(
         LibraAccount::restore_withdraw_capability(vasp_withdrawal_cap);
     };
 }
+spec fun create_child_vasp_account {
```
Comment (sblackshear) [inline language/stdlib/transaction_scripts/create_child_vasp_account.move:49]: Add ensures (if it's not too complex)?

Comment (tzakian) [reply language/stdlib/transaction_scripts/create_child_vasp_account.move:49]: 😢  Even the simplest of ensures seems to make the prove unhappy. I think once we have more specs in LibraAccount we'll be able to add some.

**Pull request closed without merging**